com.unity.multiplayer.mlapi/Tests/Runtime/NetworkManagerHelper.cs

```diff
@@ -0,0 +1,207 @@
+using System;
+using System.Collections.Generic;
+
+
+namespace MLAPI.RuntimeTests
+{
+    using UnityEngine;
+    using UnityEngine.SceneManagement;
+    using NUnit.Framework;
+    using MLAPI.SceneManagement;
+    using MLAPI.Transports.UNET;
+
+
+    /// <summary>
+    /// Helper class to instantiate a NetworkManager
+    /// This also provides the ability to:
+    /// --- instantiate GameObjects with NetworkObject components that returns a Guid for accessing it later.
+    /// --- add NetworkBehaviour components to the instantiated GameObjects
+    /// --- spawn a NetworkObject using its parent GameObject's Guid
+    /// Call StartNetworkManager in the constructor of your runtime unit test class.
+    /// Call ShutdownNetworkManager in the destructor  of your runtime unit test class.
+    ///
+    /// Includes a useful "BuffersMatch" method that allows you to compare two buffers (returns true if they match false if not)
+    /// </summary>
+    public static class NetworkManagerHelper
```

Comment:
I feel like this helper should've been on its own branch and PR, reviewed there in isolation but `¯\_(ツ)_/`

Reply:
Yeah, but I needed it to do different unit tests that all used NetworkManager.  So, this helper class allows me to do this without repeating the same code over and over within the same class.  Also, there is an issue if you Assert and have created the NetworkManager within the UnitTest itself... the Assert breaks you from the UnitTest and will ignore and "clean up" you might have done towards the end of your UnitTest.  This includes instantiating GameObjects and the like.  Sooo...so as to not submit UnitTests that leave things hanging around, I created this...and then realized it was pretty helpful (also moved the BuffersMatch into this and reference it in my tests and in some components...because it is helpful)